aws/table_aws_backup_vault.go

```diff
@@ -20,9 +20,10 @@ func tableAwsBackupVault(_ context.Context) *plugin.Table {
 		Name:        "aws_backup_vault",
 		Description: "AWS Backup Vault",
 		Get: &plugin.GetConfig{
-			KeyColumns: plugin.AnyColumn([]string{"name"}),
+			KeyColumns: plugin.SingleColumn("name"),
+			// DescribeBackupVault API returns AccessDeniedException when it is called with other regions where the vault is unavailable
```

Comment:
```suggestion
			// DescribeBackupVault API returns AccessDeniedException instead of a not found error when it is called for vaults that do not exist
```